sound/soc/sof/topology.c

```diff
@@ -494,6 +494,13 @@ static int sof_control_load_volume(struct snd_soc_component *scomp,
 	}
 
 out:
+	/* set up possible led control from mixer private data */
+	scontrol->led_ctl.use_leds = 0;
+	if (le32_to_cpu(mc->priv.size) == 1) {
```

Comment:
@juimonen we should check for priv size > 0 here instead of just 1. What is there is more data for something else

Reply:
Also, to have just one byte to indicate the use of LED and its direction is confusing. If you add more private data in the future for something else, then the first byte will automatically be there. I think we should have 2 bytes for both use_leds and direction.

Reply:
ok, updated also topology in fw pr.